salt/modules/esxcluster.py

```diff
@@ -1,5 +1,15 @@
 """
 Module used to access the esxcluster proxy connection methods
+
+.. Warning::
+    This module will be deprecated in a future release of Salt. VMware strongly
+    recommends using the
+    `VMware Salt extensions <https://docs.saltproject.io/salt/extensions/salt-ext-modules-vmware/en/latest/all.html>`_
+    instead of the ESXi module. Because the Salt extensions are newer and
```

Comment:
Since this is the esxcluster module, should this not reference the ESXCluster module.

Reply:
@dmurphy18 , thanks for your review. I have some follow-up questions: are you saying that I should instead say "VMware strongly recommends using the VMware Salt extensions instead of the ESXcluster module." Is that the fix you're wanting?

Reply:
Yes, and similarly below for the other modules.
Since it is the ESXcluser module here and stating ESXi module would lead to confusion.

Reply:
Cool. I'll make that change. Should be easy enough.